.circleci/local_publish_helpers.sh

```diff
@@ -56,7 +56,7 @@ function generatePkgCli {
 
   # install package depedencies
   cp ../yarn.lock ./
-  yarn --production
+  yarn workspaces focus --production
```

Comment:
what motivated this change?

Reply:
this: https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli/19615/workflows/b2fe5a1f-29e4-494f-9c79-f02ea51d056a/jobs/937156
cc: @evcodes